cmake/modules/RootBuildOptions.cmake

```diff
@@ -293,6 +293,10 @@ if(NOT CMAKE_CXX_STANDARD GREATER 11)
   set(webui_defvalue OFF)
 endif()
 
+if (NOT webui AND NOT "${webui}" STREQUAL "")
```

Comment:
Should this go to where it's used, line 315? It seems the combination of `if(NOT webui AND NOT webui_disabled)` with `if (NOT webui AND NOT "${webui}" STREQUAL "")` is mroe complex than needed - isn't this "simply"

```
if(NOT webui AND NOT "${webui}" STREQUAL "")
  set(webui ON CACHE BOOL "(webui built automatically when root7 is enabled)" FORCE)
endif()
```
i.e. do we really need `webui_disabled`?

Reply:
> i.e. do we really need webui_disabled?

Problem that `webui` is cmake option with own default values. 
At this place I already have no idea if user intentionally want to disable it - that's why extra variable.

Reply:
I was talking about the variable called `webui_disabled` - do we need it, or can be simply combine its `if` with the `if` below that uses `webui_disabled`?

Reply:
> I was talking about the variable called webui_disabled - do we need it, or can be simply combine its if with the if below that uses webui_disabled

No, it cannot be combined. When `webui_disabled` set, `root7` is not evaluated. 
I am not big expert in cmake. I will check if following is works:

```
set(webui_defvalue ${root7})
```
Does it work before `root7` option is evaluated?

Reply:
```
set(webui_defvalue ${root7})
```
Seems to be it works!